tests/api_data/endpoint_data.json

```diff
@@ -1,21 +1,51 @@
 {
```

Comment:
📌 Would it be better to use monkey patches in the future instead of a test dictionary? (Or probably both)

Reply:
yes in short. I find the test dictionary is (or has been) handy for locally development in that it's just an easy to maintain version of api contracts (while the apis themselves are in development). But yes, should be more formally abstracted from code